api/v1/db/oracledb/game-dao.js

```diff
@@ -0,0 +1,61 @@
+const appRoot = require('app-root-path');
+const _ = require('lodash');
+
+
+const { serializeGames } = require('../../serializers/games-serializer');
+
+const conn = appRoot.require('api/v1/db/oracledb/connection');
+const mergeRawGames = (rawGames) => {
+  const groupedRawGames = _.groupBy(rawGames, 'GAME_ID');
+  const mergedRawGames = _.map(groupedRawGames, (gameMetaDataArray) => {
+    if (gameMetaDataArray[0].CARD_NUMBER == null) {
```

Comment:
Could you use `===` here instead of `==`? This will make sure the actual value is `null` instead of something like `undefined` for example
```js
undefined == null
// true
undefined === null
// false
```